examples/afl_untracer/afl-untracer.c

```diff
@@ -56,6 +56,7 @@
 #include <sys/shm.h>
 #include <sys/wait.h>
 #include <sys/types.h>
+#include <sys/personality.h>
```

Comment:
It is Linux-only, right?

Reply:
it is using ptrace so yes it is linux only